evalml/automl/automl_search.py

```diff
@@ -614,29 +599,26 @@ def _get_mean_cv_scores_for_all_objectives(cv_data):
                     scores[field] += value
         return {objective_name: float(score) / n_folds for objective_name, score in scores.items()}
 
-    def _compute_cv_scores(self, pipeline, X, y):
+    def _compute_cv_scores(self, pipeline):
```

Comment:
It would be great to get more eyes on the changes here. I think I did it right but can't hurt to check.